src/main/java/com/google/devtools/build/lib/analysis/config/InvalidConfigurationException.java

```diff
@@ -45,17 +45,17 @@ public InvalidConfigurationException(String message, Throwable cause) {
   }
 
   public InvalidConfigurationException(Throwable cause) {
-    super(cause.getMessage(), cause);
+    super(cause == null ? null : cause.getMessage(), cause);
```

Comment:
Please put in  `TODO: https://github.com/bazelbuild/bazel/issues/24239 - find and clean up null cause`